cmd/vicadmin/vicadm.go

```diff
@@ -544,27 +575,26 @@ func (s *server) serve() error {
 	s.handleFunc("/container-logs.tar.gz", s.tarContainerLogs)
```

Comment:
I missed this one :(
I couldn't find a good solution to authenticate all requests for the service without repeating myself. I tried to find a solution but could not, and I fear that's due to lack of experience with the language. If there is a way to do this for all requests I'd love to see it, but since each HandlerFunc is defined in-place and there is no inheritance... I'm not sure that it's possible.